init/docker/docker-compose.env.example

```diff
@@ -17,4 +17,5 @@ POLLER_DEBUG=false
 POLLER_SAVE_DPI=false
 UNIFI_USER=unpoller
 UNIFI_PASS=set_this_on_your_controller
+UNIFI_APIKEY=create_this
```

Comment:
Note the way the logic works is if not empty it will prefer this, maybe leave blank?